exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/fn/JsonReader.java

```diff
@@ -108,6 +109,7 @@ public Builder(DrillBuf managedBuf) {
       this.skipOuterList = false;
       this.allTextMode = false;
       this.enableNanInf = true;
+      this.enableEscapeAnyChar = false;
```

Comment:
Lines above this are setting this to false explicitly, so to keep it consistent I set it to false. But sure, I can change it. Do I also change others to remove the explicit false assignment?

Reply:
I think i'll leave it as-is for consistency and clarity reasons - unless you think otherwise. If not, we'll need to change other options above as well.

Reply:
My suggestion would be to cleanup others as well but it's not mandatory.

Reply:
Done. I removed the explicit initialization of these variables in the constructor.

Reply:
Please check.